composer.json

```diff
@@ -8,8 +8,11 @@
     "spryker/propel-orm": "^1.0.0",
     "spryker/zend": "^2.1.0"
   },
+  "require-dev" : {
+    "spryker/synchronization": "^0.1.0 || ^0.2.0"
+  },
   "suggest": {
-    "spryker/synchronizationBehavior": "^0.1.0"
+    "spryker/synchronization": "*"
```

Comment:
This should be a text, why one would want to use it.

Reply:
I will fix that in CORE-2977